client_settings.md

```diff
@@ -72,7 +72,10 @@ There are lots of client settings available, and this doc will guide you through
 |`cl_warning_teambalance`| 	Warning if teams are unbalanced| 	1|
 |`cl_mouse_deadzone`| 	Maximum distance the mouse can be moved without the camera moving| 	400|
 |`cl_mouse_followfactor`| Set how fast the camera will follow the cursor| 	60|
-|`cl_mouse_max_distance`| 	Maximum distance between cursor and player| 	800|
+|`cl_mouse_max_distance`| 	Maximum distance between cursor and player(pre 0.7.1 setting)| 	800|
```

Comment:
This option no longer exists in Teeworlds. I don't think we should keep old options in the documentation?

Reply:
ok, sounds good :D